src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/electric/machines/ElectricIngotPulverizer.java

```diff
@@ -50,6 +51,14 @@ public ItemStack getProgressBar() {
         return displayRecipes;
     }
 
+    @Override
+    protected void registerDefaultRecipes() {
+        // this is an extra recipe on top of PostSetup.loadSmelteryRecipes() for converting Vanilla Gold Ingot to Slimefun gold dust
+        registerRecipe(3,
+                new ItemStack[] { new ItemStack(Material.GOLD_INGOT) },
+                new ItemStack[] { new ItemStack(SlimefunItems.GOLD_DUST) });
```

Comment:
No need to do `new ItemStack()` around the Slimefun Item, it would just create a wasteful and object there.
Other than that, good for merging!